### PR TITLE
fix English Doc of API:layers.array_read/array_write/array_length

### PR DESCRIPTION
**Pull request by zhwesky2010**
Because these 3 API are in the same file , so fix English Doc of them together: array_read、array_write、array_length.

**1.array_read** : This PR is the same with chinese [PR1391](https://github.com/PaddlePaddle/FluidDoc/pull/1391)

MacHi 2019-10-10 20-57-06
MacHi 2019-10-10 20-57-37

**2.array_write** : This PR is the same with chinese [PR1345](https://github.com/PaddlePaddle/FluidDoc/pull/1345)

MacHi 2019-10-10 20-58-12
MacHi 2019-10-10 20-58-24

**3.array_length** : This PR is the same with chinese [PR1344](https://github.com/PaddlePaddle/FluidDoc/pull/1344)

MacHi 2019-10-10 20-58-45
MacHi 2019-10-10 20-58-57

### CODE REVIEW

File: python/paddle/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -930,31 +930,53 @@ def increment(x, value=1.0, in_place=True):
...
-        i (Variable|list): The index of the output LOD_TENSOR_ARRAY, pointing to
-                           the position to which the input tensor will be
-                           written.
-        array (Variable|list): The output LOD_TENSOR_ARRAY to which the input
-                               tensor will be written. If this parameter is
-                               NONE, a new LOD_TENSOR_ARRAY will be created and
-                               returned.
+        x (Variable): the input data to be written into array. It's multi-dimensional
+                      Tensor or LoDTensor. data type: float32, float64, int32, int64.
+        i (Variable): 1-d Tensor with shape [1], which represents the position to which
```
Comment (hong19860320) [inline python/paddle/fluid/layers/control_flow.py]: 1-d改成大写的吧 1-D

File: python/paddle/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -930,31 +930,53 @@ def increment(x, value=1.0, in_place=True):
...
+                      ``x`` is written. data type: int64.
+        array (Variable, optional): the LoDTensorArray to which ``x`` is written.
```
Comment (hong19860320) [inline python/paddle/fluid/layers/control_flow.py]: 冒号后面的第一个字母都要大写

File: python/paddle/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -1310,29 +1360,50 @@ def shrink_memory(x, i, table):
...
+    :ref:`api_fluid_layers_While` OP to traverse, read and wirte LoDTensorArray.
 
     Args:
-        array (LOD_TENSOR_ARRAY): The input array that will be used
+        array (LoDTensorArray): The input array that will be used
                                   to compute the length.
 
-    Returns:
-        Variable: The length of the input LoDTensorArray.
+    Returns:
```
Comment (hong19860320) [inline python/paddle/fluid/layers/control_flow.py]: 去掉多余的空格吧

**Review by lanxianghit — DISMISSED** [review]

File: python/paddle/fluid/layers/control_flow.py (hunk 1 above)
Comment (zhwesky2010) [reply python/paddle/fluid/layers/control_flow.py]: Done

File: python/paddle/fluid/layers/control_flow.py (hunk 2 above)
Comment (zhwesky2010) [reply python/paddle/fluid/layers/control_flow.py]: 已更正

File: python/paddle/fluid/layers/control_flow.py (hunk 3 above)
Comment (zhwesky2010) [reply python/paddle/fluid/layers/control_flow.py]: Done

**Review by hong19860320 — DISMISSED** [review]
LGTM

**Review by hong19860320 — DISMISSED** [review]
LGTM

**Review by lanxianghit — DISMISSED** [review]

**Review by hong19860320 — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**